.gitignore

```diff
@@ -10,7 +10,6 @@ build/
 dist/
 coverage/
 scratch/
-*.d.ts
```

Comment:
Rather than do this, I'd rather just carve out an exception in the types package (as we do in the utils package). In general, these are build artifacts, and shouldn't be committed to version control.

Reply:
I added this back because this change became unnecessary with the latest changes f84a2157d6389b109b28b2d7bf0ee285a6bcc4d1